game/resource/English/ability/units/tooltip_lich_dark_ritual.txt

```diff
@@ -0,0 +1,7 @@
+//=============================================================================
+// LICH SACRIFICE
+//=============================================================================
+	"DOTA_Tooltip_ability_lich_dark_ritual"							"Sacrifice"
+	"DOTA_Tooltip_ability_lich_dark_ritual_Description"					"Sacrifices a creep and converts its current hit points into mana for Lich.\nCan be used on friendly or neutral creeps."
```

Comment:
We changed vanilla stuff right? We should color our changes. 
You could use some html
```html
Sacrifices a creep and converts its current hit points into mana for Lich.\nCan be used on friendly <font color=orange>or neutral creeps</font>.
```